packages/lodestar-cli/src/wallet/Wallet.ts

```diff
@@ -0,0 +1,113 @@
+import * as bip39 from "bip39";
+import {mapValues} from "lodash";
+import {PrivateKey} from "@chainsafe/bls";
+import {Keystore} from "@chainsafe/bls-keystore";
+import {
+  deriveEth2ValidatorKeys,
+  IEth2ValidatorKeys,
+} from "@chainsafe/bls-keygen";
+
+/**
+ * @chainsafe/bls-keystore@1.0.0-beta8 requires a pubKey argument
+ * While the library is not agnostic use this empty value 
+ */
+const emptyPubkey = Buffer.from("");
+
+export interface IWalletKeystoreJson {
+  crypto: object;
+  uuid: string;
+  name: string;
+  nextaccount: number;
+  version: number;
+  type: string;
+}
+
+export class Wallet extends Keystore {
```

Comment:
Ideally we pull this out into a bls-walletstore library

Reply:
Agree, would mind taking care of creating the library's repo and NPM package?

Reply:
:+1: Should we wait until we review https://github.com/ChainSafe/bls-keystore/pull/6 ?

Reply:
I've created an issue in https://github.com/ChainSafe/bls-keystore (for lack of a better place) to track this https://github.com/ChainSafe/bls-keystore/issues/8